application/src/modules/homepage/HomepageState.js

```diff
@@ -55,6 +66,7 @@ export default function HomepageStateReducer(state = initialState, action = {})
     case NOTIFICATION_RESPONSE:
       // We got a notification update so let's update the state and then restart the timer.
       return loop(
+        // TODO(@iprunache) stop triggering a render for every poll when the payload does not change; happens with immutable too.
```

Comment:
@iprunache just because a reducer is triggered for every fetch (which is normal), I don't think actual re-rendering takes place. More details here: https://facebook.github.io/react/docs/reconciliation.html